modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/KotlinClientCodegen.java

```diff
@@ -1114,6 +1114,22 @@ private static boolean isMultipartType(List<Map<String, String>> consumes) {
         return false;
     }
 
+    @Override
+    public void postProcessParameter(CodegenParameter parameter) {
+        super.postProcessParameter(parameter);
+        adjustEnumRefDefault(parameter);
+    }
+
+    private void adjustEnumRefDefault(CodegenParameter param) {
```

Comment:
thanks for the PR

what about adding a docstring explaining what this function does?

Reply:
i'll do it in another pr instead.